#!/bin/sh

CWD=`pwd`
GCC_VER=`basename $CWD`

cp ../org/$GCC_VER/gcc/config/mips/mips-tables.opt gcc/config/mips/mips-tables.opt
diff -drup ../org/$GCC_VER/gcc/config/mips gcc/config/mips | fgrep -v "Only in gcc/config/mips:"
diff -dup /dev/null gcc/config/mips/f32c.md
diff -dup ../org/$GCC_VER/configure configure
diff -dup ../org/$GCC_VER/gcc/doc/invoke.texi gcc/doc/invoke.texi
